From 4ac18ab3dfb025d90255c6ac74ade05c32ece6fd Mon Sep 17 00:00:00 2001
From: shengfei Xu <xsf@rock-chips.com>
Date: Wed, 25 Mar 2020 16:09:07 +0800
Subject: [PATCH] ARM: dts: rv11xx-evb-v10: fix the regulator value

the regulator-suspend-microvolt of vdd_logic is 800000uV.
the regulator-min-microvolt of vdd_arm is 800000uV

Signed-off-by: shengfei Xu <xsf@rock-chips.com>
Change-Id: I20ac4486a558d3f5562d7de03f613a002bf69511
---
 arch/arm/boot/dts/rv11xx-evb-v10.dtsi | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm/boot/dts/rv11xx-evb-v10.dtsi b/arch/arm/boot/dts/rv11xx-evb-v10.dtsi
index 425894d88234..2ad91b9c53de 100644
--- a/arch/arm/boot/dts/rv11xx-evb-v10.dtsi
+++ b/arch/arm/boot/dts/rv11xx-evb-v10.dtsi
@@ -229,14 +229,14 @@
 				regulator-name = "vdd_logic";
 				regulator-state-mem {
 					regulator-on-in-suspend;
-					regulator-suspend-microvolt = <80000>;
+					regulator-suspend-microvolt = <800000>;
 				};
 			};
 
 			vdd_arm: DCDC_REG2 {
 				regulator-always-on;
 				regulator-boot-on;
-				regulator-min-microvolt = <80000>;
+				regulator-min-microvolt = <800000>;
 				regulator-max-microvolt = <1350000>;
 				regulator-ramp-delay = <6001>;
 				regulator-initial-mode = <0x2>;
-- 
2.35.3

